internal/runners/export/log.go

```diff
@@ -85,3 +100,31 @@ func (l *Log) Run(params *LogParams) (rerr error) {
 
 	return nil
 }
+
+func ignoreLogFile(logFile string) (bool, error) {
+	file, err := os.Open(logFile)
+	if err != nil {
+		return false, errs.Wrap(err, "failed to open log file")
+	}
+	defer file.Close()
+
+	regex := regexp.MustCompile(`.*\] Args: \[(.*?)\], Flags: \[.*?\]`)
+	scanner := bufio.NewScanner(file)
+	var args string
+	for scanner.Scan() {
+		logLine := scanner.Text()
+		if regex.MatchString(logLine) {
+			match := regex.FindStringSubmatch(logLine)
+			if len(match) > 1 {
+				args = match[1]
+			}
+			break
+		}
+	}
+
+	if strings.Contains(args, "export") && strings.Contains(args, "log") {
```

Comment:
I'd prefer we be a but more explicit with the matches. However unlikely, it's easy enough to guard against something like `state exec -- myapp -i file.log --export` or something along those lines.